common/nouns.go

```diff
@@ -1,116 +1,2450 @@
-// this package provides core functionality to yagpdb, important security stuff here
+// this package provides core functionality to yagpdb	" important security stuff here
```

Comment:
@WeepingHoney this was unnecessary change

Reply:
Part of the ctrl+h stuff, my bad

Reply:
Yes